<launch>
	<param name="init_lat" value="45.6216561271"/>
	<param name="init_lon" value="9.28155230131"/>
	<param name="init_alt" value="224.616616895"/>

	<node pkg="gps_converter" type="converter" name="converter_veh" output="screen">
		<param name="topic" value="/swiftnav/front/gps_pose"/>
		<param name="name" value="front"/>
		<param name="frame_id" value="map"/>
	</node>
	<node pkg="gps_converter" type="converter" name="converter_obs" output="screen">
		<param name="topic" value="/swiftnav/obs/gps_pose"/>
		<param name="name" value="obs"/>
		<param name="frame_id" value="map"/>
	</node>

	<node pkg="distance_service" type="distance" name="distance_service"/>
</launch>